pulsar-broker/src/main/java/org/apache/pulsar/broker/web/RestException.java

```diff
@@ -80,9 +68,9 @@ private static Response getResponse(Throwable t) {
             return e.getResponse();
         } else {
             return Response
-                .status(Status.INTERNAL_SERVER_ERROR)
-                .entity(getExceptionData(t))
-                .type(MediaType.TEXT_PLAIN)
+                .status(Status.INTERNAL_SERVER_ERROR.getStatusCode(), t.getMessage())
+                .entity(new ErrorData(t.getMessage()))
```

Comment:
Should we ignore the stack trace info?

Reply:
What do you mean stack trace info ?

Reply:
I notic that the old behavior with stack trace info.

Reply:
ah, ok. yes, I removed that.